### PR TITLE
fix: Use github:bootstrap@4.0.0-alpha.2

### PR DESCRIPTION
**Pull request by jakeNiemiec**

### CODE REVIEW

**Comment by DavidStrada** [comment]
First off thank you :), I have a question, will this include lib folder, e.g. scss files  ?

File: package.json [hunk 1]
```diff
@@ -8,14 +8,43 @@
     "name": "app",
     "main": "src/app.js",
     "dependencies": {
-      "bootstrap": "npm:bootstrap@4.0.0-alpha.2",
+      "bootstrap": "github:twbs/bootstrap@4.0.0-alpha.2",
```
Comment (jakeNiemiec) [inline package.json:11]: This was the root of the problem was that `jspm i github:twbs/bootstrap@4.0.0-alpha.2` and `jspm i npm:twbs/bootstrap@4.0.0-alpha.2` get configured differently.

Comment (DavidStrada) [inline package.json:11]: I see, I also need to pull scss files how can achieve this ?    `"files": null,` ?

Comment (DavidStrada) [inline package.json:11]: this ones --> https://github.com/twbs/bootstrap/tree/v4.0.0-alpha.2/scss

Comment (jakeNiemiec) [inline package.json:11]: Looking...

Comment (DavidStrada) [inline package.json:11]: previously i was doing this `"files": [ "dist", "fonts", "js", "sass", "package.json" ],` [gist](https://gist.github.com/DavidStrada/bd33fdfdc876b4e48e2f3358fd6b66b8)

**Pull request merged**